tests/blockchain.rs

```diff
@@ -1,14 +1,15 @@
 use fuel_crypto::{Hasher, SecretKey};
-use fuel_tx::{StorageSlot, TransactionBuilder};
+use fuel_tx::TransactionBuilder;
 use fuel_types::bytes;
-use fuel_vm::script_with_data_offset;
-use fuel_vm::util::test_helpers::TestBuilder;
+// use fuel_vm::script_with_data_offset;
```

Comment:
Idk if it was the intent, but we need to indeed remove these manual offsets since now we can trivially fetch them with `GTF`. Anyway, I'm very happy to see the overall coverage improvement

Reply:
I certainly didn't intend to have this comment left behind. I removed the commented uses.